src/widgets/helper/SearchPopup.cpp

```diff
@@ -167,6 +168,7 @@ std::vector<std::unique_ptr<MessagePredicate>> SearchPopup::parsePredicates(
     auto predicates = std::vector<std::unique_ptr<MessagePredicate>>();
     auto words = input.split(' ', QString::SkipEmptyParts);
     auto authors = QStringList();
+    auto channels = QStringList();
```

Comment:
I think I would prefer `QStringList channels;` instead but keeping it consistent with the lines above is also fine by me.
(Really, the lines above should be changed probably)

Reply:
Yeah I pretty much went along with how the original change was done. As you said might as well change the lines above too, but would I do this as a part of this PR?

Reply:
Can't blame for you for code you didn't write (or for sticking to the convention established in the same file) so I think either way is fine. Just go with what you feel most comfortable with :+1: